test/bin/sandman/test_walk.py

```diff
@@ -45,18 +48,19 @@ class _DummyGroup(IdM.base.Group):
     _owner: _DummyUser
     _member: _DummyUser
 
-    def __init__(self, gid, owner, member=None):
+    def __init__(self, gid, owner, member=None, num_owners: int = 3):
         self._id = gid
         self._owner = owner
         self._member = member or owner
+        self._num_owners = num_owners
 
     @property
     def name(self):
-        raise NotImplementedError
+        return f"dummy group {self._id}"
```

Comment:
Is it possible to refactor the dummy stuff used in the different test files so they all use the same implementation, rather than re-implement these changes multiple times?